frontend/src/components/account-mgmt/ManageUserDialog.vue

```diff
@@ -122,13 +122,13 @@
       <template #button>
         <v-row justify="space-around">
           <v-col cols="12" md="6" class="d-flex justify-center">
-            <AppButton id="cancel-reply-request" :primary="false" size="large" width="200px" @click="closeManageUserDialog()" :loading="isLoading">Cancel</AppButton>
+            <AppButton id="cancel-reply-request" :primary="false" size="large" width="200px" :loading="isLoading" @click="closeManageUserDialog()">Cancel</AppButton>
           </v-col>
           <v-col cols="12" md="6" class="d-flex justify-center">
-            <AppButton v-if="!wasNewUserAdded || isUpdatingUser" id="submit-reply-request" size="large" width="200px" @click="saveUser()" :loading="isLoading">
+            <AppButton v-if="!wasNewUserAdded || isUpdatingUser" id="submit-reply-request" size="large" width="200px" :loading="isLoading" @click="saveUser()">
```

Comment:
I think we need to update the logic of doesUserExist() which checks for duplicate users having same First name, Last name, Email. After making First name optional, I am able to create duplicate user because it is passed to the query as undefined
![image](https://github.com/user-attachments/assets/beae4d13-2f6b-4af6-930e-fe09dc63721d)

Reply:
Replacing undefined with null doesn't work as it just treats it as a string. Instead we need to replace with an empty string but that means we lose are ability to do an exact match on an empty string as it's effectively ignored in the Dynamics query. Instead, I've put together the following workaround. Feel free to test and improve as necessary.